src/stories/containers/User/EditUser/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Form } from 'antd';
+import { action } from '@storybook/addon-actions';
+import { path } from 'ramda'
+import buildUser from '../../../../utils/userSpec';
```

Comment:
adicionar quebra de linha.

```suggestion
import { path } from 'ramda'

import buildUser from '../../../../utils/userSpec';
```